README.rst

```diff
@@ -0,0 +1,4 @@
+# qp-qiime2
```

Comment:
Note that by changing the file format from markdown (md) to rst, the formatting of the file has been lost (check on your branch). Can you either leave it as an md file or format it correctly?

Thanks